neurow/lib/neurow/application.ex

```diff
@@ -74,8 +74,8 @@ defmodule Neurow.Application do
          scheme: :http, plug: Neurow.InternalApi.Endpoint, options: [port: internal_api_port]},
         {Plug.Cowboy,
          scheme: sse_http_scheme, plug: Neurow.PublicApi.Endpoint, options: public_api_http_config},
-        {Plug.Cowboy.Drainer, refs: [Neurow.PublicApi.Endpoint], shutdown: 20_000},
-        {StopListener, []},
+        {Plug.Cowboy.Drainer, refs: [Neurow.PublicApi.Endpoint.HTTP], shutdown: 20_000},
```

Comment:
Discovered a regression introduced in a previous PR, prefixing by `.HTTP` fixes it.